### PR TITLE
4372 Binance configurable fetch limit

### PR DESCRIPTION
**Pull request by 0xfaulty**
Solution for issue https://github.com/knowm/XChange/issues/4372

### CODE REVIEW

File: xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
   protected static final String USE_HIGHER_UPDATE_FREQUENCY =
       "Binance_Orderbook_Use_Higher_Frequency";
   protected static final String USE_REALTIME_BOOK_TICKER = "Binance_Ticker_Use_Realtime";
+  protected static final String USE_MAX_FETCH_ORDER_BOOK_LIMIT = "Binance_Max_Fetch_Order_Book_Limit";
```
Comment (walec51) [inline xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java:30]: the words "use" and "max" make little sense for this parameter

just name it

FETCH_ORDER_BOOK_LIMIT = "Binance_Fetch_Order_Book_Limit"

**Review by walec51 — COMMENTED** [review]
looks ok

just a small naming remark

File: xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java (hunk 1 above)
Comment (0xfaulty) [reply xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java:30]: Does this mean that this parameter will no longer be Boolean, but will become an Integer with a specific limit value? Do I need to validate it for valid values (5, 10, 20, 50, 100, 500, 1000, 5000) in this case, or is the correctness of the value the responsibility of the api user?

Comment (0xfaulty) [reply xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingExchange.java]: @walec51 to speed up the acceptance of the PR, decided to redo it before your answer. Renamed parameter name and used only if integer inside parameter.

**Pull request merged**